tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs

```diff
@@ -145,4 +150,40 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
 
     /// <inheritdoc/>
     protected override void LogToolCommand(string message) => Log.LogMessage(MessageImportance.Normal, message);
+
+    /// <inheritdoc/>
+    protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
What is the default ExecuteTool implementation that we override here?

Reply:
https://github.com/dotnet/msbuild/blob/b8e5cce93e2e664eaf470c0fed9f5b84b8591e1e/src/Utilities/ToolTask.cs#L715

Reply:
I'm not sure that we need to override this method. The base implementation does many things see:
 https://github.com/dotnet/msbuild/blob/b8e5cce93e2e664eaf470c0fed9f5b84b8591e1e/src/Utilities/ToolTask.cs#L715

Why don't we process the messages in https://github.com/icerpc/icerpc-csharp/blob/0153bc8323107da94d1fc9a360f59d737647b480/tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs#L93

MSBuild calls LogEventsFromTextOutput to process each line sent to stderr or stdout.  The trick here is that we either get all the output on a single line, accumulate it and retry.

Reply:
This sounds like a good idea, but I don't understand the difficulty. 

Does slicec-cs output anything else to stdout? Warnings and errors go to stderr, don't they?

Reply:
I think that with this override `LogEventsFromTextOutput` would not be called, and errors would not be properly logged, to be confirmed.

Reply:
> This sounds like a good idea, but I don't understand the difficulty.

It shouldn't be to difficult. We are already using this method to process error messages. It is simpler when the compiler emits a complete JSON message in each line, and the task can parse it right away and decide what to do with it.